CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Pythie serving
 
+## 3.0.0.dev1
+## Change
+* Add GRPC server timeout 
```

Comment:
can you add the name of the variable to setup and put it the README too? (there's a section about available env variables)

Reply:
+ we can release as 3.1.0 now that you have tested it